velox/functions/sparksql/LeastGreatest.cpp

```diff
@@ -155,6 +156,13 @@ std::vector<std::shared_ptr<exec::FunctionSignature>> leastSignatures() {
                                 .variableArity()
                                 .build());
   }
+  signatures.emplace_back(exec::FunctionSignatureBuilder()
+                              .integerVariable("p")
+                              .integerVariable("s")
+                              .returnType("decimal(p,s)")
+                              .argumentType("decimal(p,s)")
+                              .variableArity()
```

Comment:
As this function requires at least two arguments, I think an extra `.argumentType("decimal(p,s)")` should be added.

Quote from scalar-functions.rst:
_"FunctionSignatureBuilder allows specifying that the last augment may appear zero or more times by calling variableArity() method"_

Reply:
Then we need to repeat argumentType for other types also.

Reply:
Good catch, Updated, thanks.